common/utils/utils.go

```diff
@@ -521,11 +521,12 @@ func ReadSpannerSchema(ctx context.Context, conv *internal.Conv, client *sp.Clie
 		conv.Unexpected(fmt.Sprintf("error trying to fetch interleave table info from schema: %v", err))
 	}
 	// Assign parents if any.
+	// TODO: Assign ParentTable.OnDelete too
```

Comment:
why have we marked this as todo and not added it here? Same comment for the other TODO